nym-node-status-agent/Cargo.toml

```diff
@@ -16,12 +16,20 @@ readme.workspace = true
 
 [dependencies]
 anyhow = { workspace = true}
+bincode = { workspace = true }
+chrono = { workspace = true }
```

Comment:
plz let's not use `chrono`, it caused us enough headache, if possible use `time` like we do everywhere else

Reply:
as discussed in private, not a change for this PR but I agree.